libraries/ui-library/src/components/six-card/six-card.scss

```diff
@@ -2,8 +2,7 @@
 
 :host {
   display: inline-block;
-  box-shadow: var(--six-elevation-8dp);
-  border-radius: 0;
-  padding: 1rem;
-  margin-bottom: 0.6rem;
+  box-shadow: var(--six-elevation-4dp);
+  border: 1px solid var(--six-color-web-rock-100);
+  padding: var(--six-spacing-medium);
```

Comment:
How is this related to the six-select improvements?

Reply:
See https://github.com/six-group/six-webcomponents/pull/171#discussion_r1457531792